imports/plugins/included/default-theme/client/styles/tagGroup.less

```diff
@@ -55,8 +55,8 @@
 .rui.grouptag > .header .rui.tag.edit.create {
   text-transform: none;
 
-  input, button {
-    background-color: @tag-group-background-color;
+  input, button, .btn {
+    background-color: @tag-group-background-color !important;
```

Comment:
I tried not to use it.
I added !important here because of an un-needed inline background style that gets added to "react-autosuggest__container" changing the box color. Couldn't find a better/faster approach